src/applications/caregivers/components/ConfirmationPage/ConfirmationScreenView.jsx

```diff
@@ -27,7 +27,7 @@ const ConfirmationScreenView = ({ form, name, timestamp }) => {
       </div>
 
       <va-alert status="info" class="vads-u-margin-bottom--4" background-only>
-        <h2 slot="headline" className="vads-u-font-size--h3">
+        <h2 className="vads-u-font-size--h3 vads-u-margin-bottom-2">
```

Comment:
removed the `slot`?

Reply:
Slots are hidden with background-only alerts. Need that headline in place.

Reply:
Merge issue?